src/code/z_actor.c

```diff
@@ -2867,7 +2870,7 @@ Actor* Actor_Delete(ActorContext* actorCtx, Actor* actor, GlobalContext* globalC
     func_800F89E8(&actor->projectedPos);
     Actor_Destroy(actor, globalCtx);
 
-    newFirstActor = Actor_RemoveFromTypeList(globalCtx, actorCtx, actor);
+    newFirstActor = Actor_RemoveFromCategoryList(globalCtx, actorCtx, actor);
```

Comment:
This is another variable that should probably be renamed
```suggestion
    newHead = Actor_RemoveFromCategoryList(globalCtx, actorCtx, actor);
```